app/javascript/src/LoginPage/utils/formValidation.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import validate from 'validate.js'
+
+const constraintsTypes = {
+  text: {
+    presence: true,
+    length: {minimum: 1}
+  },
+  email: {
+    presence: true,
+    email: true,
+    length: {minimum: 1}
+  },
+  password: {
+    presence: true,
+    length: {minimum: 6}
```

Comment:
So this is validating in JS + in Ruby too when the form is submitted right?
It is duplicated there https://github.com/3scale/porta/blob/master/app/models/user.rb#L80
OK for now, but I think this kind of thing will hit us really hard when it gets complicated with logic in both views and in backend

Reply:
We'll handle that in a future task: https://issues.jboss.org/browse/THREESCALE-3352

Reply:
Good for me 👍